### PR TITLE
adding small enhancements, adding basic idea for unit tests

### PR DESCRIPTION
**Pull request by dev7ch**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+// One basic idea to write unit tests ...
+
+var assert = require('assert');
+
+var getSilvan = function(name) {
```
Comment (nadar) [inline test/test.js]: crazy test :-)

**Comment by dev7ch** [comment]
unit testing isn´t complete, it needs further work

**Comment by nadar** [comment]
now it seems to test something already, which is great!

**Comment by dev7ch** [comment]
unit testing ist working good now, including deletion of previously generated test files, juchee :)

**Pull request closed without merging**